xdf2fieldtrip.m

```diff
@@ -42,18 +44,50 @@
 % $Id$
 
 % process the options
-streamindx = ft_getopt(varargin, 'streamindx');
+streamindx      = ft_getopt(varargin, 'streamindx');
```

Comment:
it seems to me that these ways of indexing the XDF file are mutually exclusive, right? That is fine with me, but you may want to add a check  to ensure that the user knows that as well. Like

````
% not more than one cal be selected
assert(sum(cellfun(@isempty, {streamindx, streamkeywords, sraterange}))<=1);
````

Reply:
Fixed - I can imagine people using two or more methods in conjunction but for now it is restricted to be exclusive for simplicity.